app/controllers/catalog_controller.rb

```diff
@@ -318,11 +318,6 @@ class CatalogController < ApplicationController
                                                      (Settings.METADATA_SHOWN &
                                                       options[:document].references.refs.map { |x| x.type.to_s }).any?
                                                  }
-    config.add_show_tools_partial :searchworks_url, component: Earthworks::SearchworksUrl,
-                                                    if: proc { |_context, _config, options|
-                                                          options[:document] &&
-                                                            options[:document].searchworks_url.present?
-                                                        }
```

Comment:
this is now replaced by the new component for the sidebar, which is also more flexible in that it allows additional links to be added in later